demo/admin/server/package.json

```diff
@@ -1,6 +1,7 @@
 {
-    "name": "server",
+    "name": "comet-demo-admin-server",
```

Comment:
Changesets always tried to publish `server@1.0.0`. Luckily for use, this version is already published on NPM 😁